client/src/reducers/game.reducer.js

```diff
@@ -14,7 +14,7 @@ export function game(state = {}, action) {
 
 
         case gameConstants.BOTS_ADDED:
-            return { ...state, players: [...state.players, ...action.bots] }
+            return { ...state, players: [...state.players, ...action.bots], bots: action.bots }
```

Comment:
Game contains bots two times? Is it what we expect ?

Reply:
Good point! Done